_posts/2023-06-20-resume-parsing-insights-and-steps-to-create-your-own-parser.md

```diff
@@ -0,0 +1,900 @@
+---
+layout: post
+title: "Resume Parsing: Insights and Steps to Create Your Own Parser"
+tldr: "We will be using NLP techniques to extract information from resume."
+modified: 2023-06-20 13:40:00 +0530
+category: technology
+tags: [Resume-parser, Python, Flask, NLP, LLM]
+author: madhav
+image:
+  feature: 
+  credit: 
+  creditlink: 
+comments: 
+share: 
+---
+
+## What is Resume Parsing
+
+Resume parsing is the automated process of extracting relevant information from resumes or CVs. 
+It analyzes the unstructured text of a resume and extracts specific details like contact information, work experience, education, skills, and achievements. 
+The extracted data is then converted into a structured format, allowing for easy analysis and integration into recruitment systems.
+
+## Benefits of Resume Parsing
+
+- It is a time-saving automation
+- It increases efficiency in candidate screening
+- Improves accuracy in data extraction
+- It standardizes the data extraction and formatting
+- It is scalable for handling large volumes of resumes
```

Comment:
Remove this line as this solution is not at all scalable. We are not doing anything in background.

Eveything is running in web request. If there will be 20 resumes at a same time. Server will crash

Reply:
Okay will remove it.